src/frontend/src/lib/components/icons/IconGitHub.svelte

```diff
@@ -1,16 +1,19 @@
-<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
-	<path
-		d="M13.4167 18V15.7071C13.4479 15.3261 13.3943 14.9431 13.2592 14.5836C13.1241 14.2241 12.9108 13.8962 12.6334 13.6219C15.25 13.3422 18 12.3915 18 8.02935C17.9997 6.91391 17.5523 5.84125 16.75 5.03337C17.1298 4.05749 17.103 2.9788 16.675 2.02141C16.675 2.02141 15.6917 1.74178 13.4167 3.20382C11.5067 2.70754 9.49337 2.70754 7.58338 3.20382C5.30839 1.74178 4.32506 2.02141 4.32506 2.02141C3.89703 2.9788 3.87018 4.05749 4.25006 5.03337C3.44183 5.84724 2.99383 6.9296 3.00006 8.05332C3.00006 12.3835 5.75004 13.3343 8.3667 13.6458C8.09253 13.9175 7.88109 14.2414 7.74612 14.5965C7.61117 14.9516 7.55571 15.3301 7.58338 15.7071V18"
-		stroke="currentColor"
-		stroke-width="1.5"
-		stroke-linecap="round"
-		stroke-linejoin="round"
-	/>
-	<path
-		d="M7 16.6756C4.85714 17.5436 3.07143 16.6756 2 14"
-		stroke="currentColor"
-		stroke-width="1.5"
-		stroke-linecap="round"
-		stroke-linejoin="round"
-	/>
+<script lang="ts">
+	export let size = '20';
+</script>
+
+<svg width={size} height={size} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+	<g clip-path="url(#clip0_2345_9164)">
+		<path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
+			d="M12 0C5.3724 0 0 5.3808 0 12.0204C0 17.3304 3.438 21.8364 8.2068 23.4252C8.8068 23.5356 9.0252 23.1648 9.0252 22.8456C9.0252 22.5612 9.0156 21.804 9.0096 20.802C5.6712 21.528 4.9668 19.1904 4.9668 19.1904C4.422 17.8008 3.6348 17.4312 3.6348 17.4312C2.5452 16.6872 3.7176 16.7016 3.7176 16.7016C4.9212 16.7856 5.5548 17.94 5.5548 17.94C6.6252 19.776 8.364 19.2456 9.0468 18.9384C9.1572 18.162 9.4668 17.6328 9.81 17.3328C7.146 17.0292 4.344 15.9972 4.344 11.3916C4.344 10.08 4.812 9.006 5.5788 8.166C5.4552 7.8624 5.0436 6.6396 5.6964 4.986C5.6964 4.986 6.7044 4.662 8.9964 6.2172C9.97532 5.95022 10.9853 5.81423 12 5.8128C13.02 5.8176 14.046 5.9508 15.0048 6.2172C17.2956 4.662 18.3012 4.9848 18.3012 4.9848C18.9564 6.6396 18.5436 7.8624 18.4212 8.166C19.1892 9.006 19.6548 10.08 19.6548 11.3916C19.6548 16.0092 16.848 17.0256 14.1756 17.3232C14.6064 17.694 14.9892 18.4272 14.9892 19.5492C14.9892 21.1548 14.9748 22.452 14.9748 22.8456C14.9748 23.1672 15.1908 23.5416 15.8004 23.424C18.19 22.6225 20.2672 21.0904 21.7386 19.0441C23.2099 16.9977 24.001 14.5408 24 12.0204C24 5.3808 18.6264 0 12 0Z"
+			fill="currentColor"
+		/>
+	</g>
+	<defs>
+		<clipPath id="clip0_2345_9164">
+			<rect width="24" height="24" fill="white" />
+		</clipPath>
+	</defs>
```

Comment:
what is this additional `defs` part?

Reply:
You are right, the `defs` part is not used